wiki/content/migration/migrate-dgraph-1-1.md

```diff
@@ -59,17 +84,31 @@ type Person {
 }
 ```
 
-After that, the node is associated with the type by adding the following RDF triple to Dgraph (using a mutation):
+Next, associate a node with the `Person` type by adding the following RDF triple
+to Dgraph (using a mutation):
```

Comment:
Comment for reviewers: This example would be a lot better if we provided an actual example mutation syntax. 

But more urgently: does the reader need to do this type of RDF mutation with all nodes, or only one node per type?

Reply:
This is required when you want to use `expand ()` queries on a given node, so I would say one node per type - @danielmai please correct me

Regarding the example, I guess the actual one is good as you can send mutations in many ways, e.g. from ratel that would would look like:
```
{
  set {
    
  <0xff> <dgraph.type> "Person" .
      }
}
```